src/components/ViewerBuildList.js

```diff
@@ -92,7 +91,9 @@ class ViewerBuildList extends React.Component {
         </TableCell>
         <TableCell className={classes.cell}>
           <div className="card-body">
-            <ReactMarkdown className="card-text" source={build.changeMessageTitle}/>
+            <Typography variant="p" color="inherit">
```

Comment:
I don't think there is such variant. You can probably just use `h6`.

Reply:
addressed